amplify-tools/gradle-mvn-push.gradle

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2013 Chris Banes
```

Comment:
It is a little troubling to watch a second copy of this script come into the git repository. Is this identically the same as the `configuration/publishing.gradle` in the main project?

I don't think there's anything ~reasonable we can do to prevent it, unfortunately. 

We _can_ make sure that the files are identical in contents and name, thought.

This one _at least_ has a different name.

Reply:
We *could* symlink it, no?

Reply:
Yea, I guess I like that better. It's ~a little weird, but it solves the problem.

Reply:
We would be able to use the ```configuration/publishing.gradle``` if not for the publishing block that seems to require an aar artifact. I think it makes sense to make the file identical in name and content except for that block